sprites.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 # Copyright (c) 2007-8, Playful Invention Company.
-# Copyright (c) 2008-14 Walter Bender
+# Copyright (c) 2008-11 Walter Bender
```

Comment:
Why change the date?

Reply:
It was changes by mistake in the previous commit, so I corrected it back to what it was previously.

Reply:
Okay.

Reply:
corrected